docs/docs/artifact-store.md

```diff
@@ -4,39 +4,38 @@ title: Artifact Store
 
 # Artifact Store
 
-Deployments inside a distributed system need a location, where application specific resources can be found.
-In Marathon we call this place the artifact store.
+Deployments inside a distributed system need a location where application-specific resources can be found. In Marathon, we call this place the artifact store.
 
 
 ## Usage
 
-The use of the Marathon Artifact Store functionality is completely optional.
-All functionality and goals can be accomplished manually as well.
-Marathon tries to simplify several use cases.
+Use of the Marathon artifact store is completely optional: you can create all of the artifact atore functionality manually as well. The Marathon artifact store simplifies several use cases.
 
 
 ## Artifact Store Backend
 
-Marathon supports different storage system, that can be used as artifact store.
-The type of the artifact store is configured via the command line.
-Example: start with --artifact_store hdfs://localhost:54310/path/to/store to use Hadoop DFS 
-as artifact storage backend.
+Marathon supports several different storage system backends for the artifact store.
```

Comment:
run Marathon --help and see the options in  --artifact_store. Currently supported:
```
hdfs, file
```